core/src/main/java/org/elasticsearch/bootstrap/JarHell.java

```diff
@@ -264,6 +264,11 @@ static void checkClass(Map<String,Path> clazzes, String clazz, Path jarpath) {
                 if (clazz.startsWith("org.apache.xmlbeans")) {
                     return; // https://issues.apache.org/jira/browse/XMLBEANS-499
                 }
+                if (clazz.startsWith("javax.xml.bind") ||
+                    clazz.startsWith("javax.activation") ||
+                    clazz.startsWith("com.sun.activation.registries")) {
+                    return; // java 9 removed from default modules ...
+                }
```

Comment:
this change isn't correct. this is for duplicate classes within the same jar. if that is happening it is indicative of some other problem (e.g. bug in the build)